CoEpi/ui/onboarding/OnboardingViewController.swift

```diff
@@ -12,8 +20,91 @@ class OnboardingViewController: UIViewController {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupGradient()
+        setupButtons()
+        setupLabels()
+    }
+    
+    func setupGradient() {
+        guard let view = view as? GradientView else { return }
+        view.startColor = Constants.color.coepiBlue
+        view.endColor = Constants.color.coepiAqua
+        view.endLocation = 0.4
+    }
+    
+    func setupButtons() {
+        privacyButton?.setAttributedTitle(privacyButtonTitle, for: .normal)
```

Comment:
If you declare the outlets as implicitly wrapped optionals, you don't need to use `?` here.

Reply:
The unwrapping was added to prevent a crash in the Unit Test target, I'm not familiar with the way the view controllers are setup / how to get a fully initialised one for use in tests. So until I work that out optional chaining prevents the crash.

Reply:
I ran the unit tests and it didn't crash. Some points:
- We shouldn't be unit testing view controllers anyway IMO. Unit Tests make more sense for business logic.
- If it crashes it would be interesting to figure out anyway. Can you check again?
- In case we really needed to use `?`, the variables should be declared as optionals, to be consistent.
- I'll remove the `?` since it's not crashing and folks want a build. Let's come back to it if it crashes for you.